  0%|                                                                                                                                                                | 0/10 [00:01<?, ?it/s]
Data(x=[11758, 10], edge_index=[2, 1038000], y=[11758], train_mask=[10, 11758], val_mask=[10, 11758], test_mask=[10, 11758], name='tolokers', random_walk_pe=[11758, 0])
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/alunos/Dados/andre/cooperative-sheaves/neural-sheaf-diffusion/cooperative_sheaves/exp/run.py", line 201, in <module>
    test_acc, best_val_acc, keep_running = run_exp(wandb.config, dataset, model_cls, fold)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alunos/Dados/andre/cooperative-sheaves/neural-sheaf-diffusion/cooperative_sheaves/exp/run.py", line 109, in run_exp
    train_loss, val_loss, tmp_test_loss] = test(model, data, fold)
                                           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alunos/Dados/andre/cooperative-sheaves/neural-sheaf-diffusion/cooperative_sheaves/exp/run.py", line 64, in test
    loss = F.binary_cross_entropy(pred)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: binary_cross_entropy() missing 1 required positional argument: 'target'
